server/routes/api/team.ts

```diff
@@ -60,4 +62,56 @@ router.post("team.update", auth(), async (ctx) => {
   };
 });
 
+router.post("team.create", auth(), async (ctx) => {
```

Comment:
🥦 creates a new team with the same auth providers as the current team, and creates a user with the same email as the current user as the admin of the new team

Reply:
Existing endpoint is singular, but it's the only one that's like that – I guess the thought was there would only ever be one team in auth scope. This is probably the moment to make it `teams.create`, I can update and address backwards compat of the other endpoints separately.